amqp/src/main/scala/akka/stream/alpakka/amqp/javadsl/AmqpSource.scala

```diff
@@ -13,6 +13,31 @@ object AmqpSource {
    * Java API: Creates an [[AmqpSource]] with given settings and buffer size.
    */
   def create(settings: AmqpSourceSettings, bufferSize: Int): Source[IncomingMessage, NotUsed] =
+    atMostOnceSource(settings, bufferSize)
+
+  /**
+   * Java API: Convenience for "at-most once delivery" semantics. Each message is acked to Kafka
+   * before it is emitted downstream.
+   */
+  def atMostOnceSource(settings: AmqpSourceSettings, bufferSize: Int): Source[IncomingMessage, NotUsed] =
+    committableSource(settings, bufferSize)
+      .map(cm => {
```

Comment:
I tried:
```
      .mapAsync(1)(cm => {
        Future {
          println(s"Committing ${cm.message.bytes.utf8String}")
          cm.ack()
          cm.message
        } (ExecutionContexts.sameThreadExecutionContext)
      })
```
But that makes the `not ack messages unless they get consumed` test fail. For some reason it ack the second element in `subscriber` even though it doesn't fetch it, and then the element is missing on `subscriber2`.
Maybe you have some idea of why is this happening?

In any case, the `ack` method of RabbitMQ library is blocking so it should be pretty much the same either way.